baggers.js

```diff
@@ -13352,39 +13352,67 @@ var data = {
         "mail": "fpot@lucca.fr"
       }
     },
-    {
```

Comment:
Why this changed? Could you please just add your bio without changing the format of other bio?